packages/backend/src/services/warehouseClients/SnowflakeWarehouseClient.ts

```diff
@@ -66,4 +131,38 @@ export default class SnowflakeWarehouseClient implements QueryRunner {
     async test(): Promise<void> {
         await this.runQuery('SELECT 1');
     }
+
+    async getSchema(
+        config: {
+            database: string;
+            schema: string;
+            table: string;
+            columns: string[];
+        }[],
+    ) {
+        const sqlText = 'SHOW COLUMNS';
```

Comment:
I guess this could be huge? If we hit performance issues in future we could apply filtering here.